build/charts/antrea-windows/README.md

```diff
@@ -0,0 +1,25 @@
+# antrea-windows
+
+![Version: 0.0.0](https://img.shields.io/badge/Version-0.0.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: latest](https://img.shields.io/badge/AppVersion-latest-informational?style=flat-square)
+
+Kubernetes networking based on Open vSwitch for Windows Nodes
+
+**Homepage:** <https://antrea.io/>
+
+## Source Code
+
+* <https://github.com/antrea-io/antrea>
+
+## Requirements
+
+Kubernetes: `>= 1.19.0-0`
```

Comment:
@shikharish @antoninbas Could you explain why 1.19 is set as the minimum required version? Is it because the versions of Helm and Kubernetes are related?

Reply:
@XinShuYang please check this issue for the discussion. We changed the minimum supported K8s to v1.19 in Antrea since 2.0.

Reply:
> @XinShuYang please check this issue #5879 for the discussion. We changed the minimum supported K8s to v1.19 in Antrea since 2.0.

Since we have removed windows docker support, the windows container manifest can only be deployed when windows host process feature is enabled. But as I know this feature was first introduced in kubernetes 1.22 (https://kubernetes.io/blog/2021/08/16/windows-hostprocess-containers/) Maybe we should update the minimum supported version for antrea windows? Do you have any suggestion? @antoninbas @luolanzone @wenyingd

Reply:
That's a good point. @shikharish please update the version requirement to 1.22 for this chart (and this chart only), and re-generate the Readme